### DISCUSSION TITLE
Error when using on Proton AOSP

### DISCUSSION THREAD

**Discussion by People-11** [Mygod/VPNHotspot#290] [Q&A]
Unable to access system hotspot settings 
Unable to instantiate Application():java.lang.RuntimeException: Unable to instantiate application android.app.Application: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference

**Comment by Mygod** [comment]
I do not guarantee support for custom ROM but could you export debug information for me?

**Reply by People-11** [reply]
Sorry for late reply, here is the information
The log file is 900k large and GitHub can't process it. So I found a [Website](https://file.io/Rj5Zll37F6Tq)  to share it

**Reply by Mygod** [reply]
Thanks for the log. I am afraid this feature is not supported on your ROM but the core functionalities of this app should not be impacted?

**Reply by People-11** [reply]
> Thanks for the log. I am afraid this feature is not supported on your ROM but the core functionalities of this app should not be impacted?

That's OK, turning off Tethering hardware acceleration can let Hotspot works fine. But still can't adjust system settings

**Reply by Mygod** [reply]
You can adjust system settings via the system settings app instead.

**Reply by People-11** [reply]
It's the ROM's issue, now it works fine, thanks.
